source/pmc/community-growth.md

```diff
@@ -3,90 +3,90 @@ title: Community Growth
 tags: ["pmc","committers","community"]
 ---
 
-Growing your project community takes work. This document makes practical
-suggestions of how to encourage people to get involved in your project.
+Building a thriving project community requires intentional effort.
+This guide provides practical strategies to attract and engage
+contributors in your project.
 
 {{% toc %}}
 
 ## Publish a roadmap
 
-Volunteers can work on whatever they want, and so we have a tendency to
-not want to tell them what to work on. However, publishing a roadmap, or
-even a wishlist of features or improvements that are desired, can give
-people an idea of where they might be able to get involved.
+While volunteers choose their own contributions, providing direction
+helps them find meaningful ways to get involved. Publishing a roadmap
+or wishlist of desired features and improvements gives potential
+contributors clear entry points.
 
-This also gives a clear sense that the project has ambitions, and is
-going somewhere, rather than that it is stagnating.
+A roadmap demonstrates that your project has clear goals and
+momentum, signaling growth rather than stagnation.
 
-It can be very challenging to keep a roadmap current. Having a mechanism
-for tagging open issues with `proposal` or `roadmap` and
-automatically extracting that into a web page can help with this.
+Keeping a roadmap current can be challenging. Consider implementing
+a mechanism for tagging open issues with `proposal` or `roadmap`
+labels and automatically extracting them into a web page to
+streamline this process.
 
 ## Periodically publish the open issue list
 
-Your ticket tracker allows you to periodically summarize the open
-issues. Sending this to the dev@ list reminds people that there are
-things to work on, and gives them permission, and encouragement, to do
-so.
+Regularly share summaries of open issues from your ticket tracker
```

Comment:
I would say "If you can, it is good to regularly share..." 

I don't think that's part of the formal PMC chair requirements, its just a nice thing to do.